common/lib/utils/region_utils.ts

```diff
@@ -0,0 +1,102 @@
+/*
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ 
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+  http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { RdsUtils } from "./rds_utils";
+import { AwsWrapperError } from "./errors";
+import { Messages } from "./messages";
+
+export class RegionUtils {
+  static readonly REGIONS: string[] = [
```

Comment:
"cn-northwest-1", "ap-southeast-7", "mx-central-1".
How is this list ordered? If there are no particular ordering can we order this alphabetically?